Userland/Libraries/LibVideo/VP9/Decoder.cpp

```diff
@@ -1406,7 +1405,7 @@ inline DecoderErrorOr<void> Decoder::inverse_discrete_cosine_transform_array_per
 
     // 1.1. A temporary array named copyT is set equal to T.
     Array<Intermediate, block_size> data_copy;
-    AK::TypedTransfer<Intermediate>::copy(data_copy.data(), data.data(), block_size);
+    memcpy(data_copy.data(), data.data(), block_size * sizeof(Intermediate));
```

Comment:
I believe `AK::Array` supports copy construction, so there is no need for this dance.

```suggestion
Array<Intermediate, block_size> data_copy = data;
```